LayoutTests/http/tests/site-isolation/accessibility/hit-test-resolving-remote-frame.html

```diff
@@ -0,0 +1,38 @@
+<html><!-- webkit-test-runner [ SiteIsolationEnabled=true ] -->
+<head>
+<script src="/js-test-resources/js-test.js"></script>
+<style>
+</style>
+</head>
+<body id="body">
+
+<iframe id="iframe" onload="runTest()" src="http://localhost:8000/site-isolation/resources/iframe.html"></iframe>
+
+<script>
+var output = "This test ensures that hit testing with site isolation enabled returns the right accessibility element when the remote element is resolved.\n\n";
+
+function runTest() {
+    if (window.accessibilityController) {
+        window.jsTestIsAsync = true;
+
+        const iframe = document.getElementById("iframe");
+        const iframeRect = iframe.getBoundingClientRect();
+
+        output += `iFrame rect: ${iframeRect.x}, ${iframeRect.y}\n`;
+
+        var text;
```

Comment:
Is this used?

Reply:
Nope, I can remove this.

Reply:
We also don't need to declare `axElement` below this, now that the test structure has changed. I'll update that as well.